tests/cmd/stability/main.go

```diff
@@ -292,6 +292,28 @@ func run() {
 				fta.StartKubeControllerManagerOrDie(vNode.IP)
 			}
 		}
+
+		// stop one kube-apiserver pod
+		faultApiServer := tests.SelectNode(cfg.APIServers)
+		fta.StopKubeAPIServerOrDie(faultApiServer)
+		defer fta.StartKubeAPIServerOrDie(faultApiServer)
+		time.Sleep(3 * time.Minute)
+		oa.CheckOneApiserverDownOrDie(ocfg, clusters, faultApiServer)
+		fta.StartKubeAPIServerOrDie(faultApiServer)
+
+		time.Sleep(time.Minute)
+		// stop all kube-apiserver pods
+		for _, physicalNode := range cfg.APIServers {
+			for _, vNode := range physicalNode.Nodes {
+				fta.StopKubeAPIServerOrDie(vNode.IP)
+			}
+		}
+		oa.CheckAllApiserverDownOrDie(ocfg, clusters)
```

Comment:
You should ensure all pods not recreated after stopping all apiservers, just like: https://github.com/pingcap/tidb-operator/pull/955/files#diff-67dacabf080ca4dc8e95d2acf12ee36bR489

Reply:
the pr：https://github.com/pingcap/tidb-operator/pull/955 have not merge. 
In my opinions, we can add this checkpoint in new PR.